docs/rules/comma-style.md

```diff
@@ -8,23 +8,33 @@ In case linting is turned off, missing commas in variable declarations lead to l
 
 ## Rule Details
 
-This rule is aimed at enforcing a particular comma style in JavaScript. As such, it warns whenever it sees a variable declaration, object property and array element that does not adhere to a particular comma style. It doesn't support cases where there are line breaks before and after comma (lone commas) with in declarations, properties and elements. It also avoids single line declaration cases.
+This rule is aimed at enforcing a particular comma style in JavaScript. As such, it warns whenever it sees a variable declaration, object property and array element that does not adhere to a particular comma style.
```

Comment:
Shorter is sweeter, in my option. :+1: 

I'm glad you mentioned that, as the use of "warn" in that context has bugged me in the past because it does confuse what should be "report" with the actual "warn" severity. As we simplify the docs to be more concise, it might not be a problem anymore, but I prefer the use of "report" over "warn" if there ends up being a sentence that still uses it.